tutorials/Atmos/burgers_single_stack.jl

```diff
@@ -2,21 +2,25 @@
 
 # Equations solved in balance law form:
 
-# ``
-# \frac{∂ ρ}{∂ t} = - ∇ ⋅ (ρu)
-# \frac{∂ ρu}{∂ t} = - ∇ ⋅ (-μ ∇u) - ∇ ⋅ (ρu u') - γ[ (ρu-ρ̄ū) - (ρu-ρ̄ū)⋅ẑ ẑ]
-# \frac{∂ ρcT}{∂ t} = - ∇ ⋅ (-α ∇ρcT) - ∇ ⋅ (u ρcT)
-# ``
+# ```math
+# \begin{align}
+# \frac{∂ ρ}{∂ t} =& - ∇ ⋅ (ρu) \\
+# \frac{∂ ρu}{∂ t} =& - ∇ ⋅ (-μ ∇u) - ∇ ⋅ (ρu u') - γ[ (ρu-ρ̄ū) - (ρu-ρ̄ū)⋅ẑ ẑ] \\
+# \frac{∂ ρcT}{∂ t} =& - ∇ ⋅ (-α ∇ρcT) - ∇ ⋅ (u ρcT)
+# \end{align}
+# ```
 
 # Boundary conditions:
-# ``
-# z_{\mathrm{min}}: ρ = 1
-# z_{\mathrm{min}}: ρu = 0
-# z_{\mathrm{min}}: ρcT = T=T_{\mathrm{fixed}}
-# z_{\mathrm{max}}: ρ = 1
-# z_{\mathrm{max}}: ρu = 0
-# z_{\mathrm{max}}: ρcT = \mathrm{no flux}
-# ``
+# ```math
+# \begin{align}
+# z_{\mathrm{min}}: & ρ = 1 \\
+# z_{\mathrm{min}}: & ρu = 0 \\
+# z_{\mathrm{min}}: & T = T_{\mathrm{fixed}}/(ρc) \\
```

Comment:
This is numerically correct in our case (\rho c = 1), but should we remove \rho c for units consistency?

Reply:
Agreed. Is it alright if we write this as `ρcT = ρc T_{\mathrm{fixed}}`?